src/main/java/com/suse/saltstack/netapi/datatypes/Token.java

```diff
@@ -31,4 +34,8 @@ public Date getStart() {
     public Date getExpire() {
         return expire;
     }
+
+    public List<String> getPerms() {
```

Comment:
well, neither of the other methods do have a javadoc. I thought it's the convention for these kind of data holder classes.
about the "perms", I wasn't converted it into "permissons" as com.suse.saltstack.netapi.datatypes.Job also have "jid", not "jobID". So I thought it is the convention to let the parameters' name as they are in the json.